blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/serializers/JSONBasicRollupsOutputSerializer.java

```diff
@@ -169,8 +169,8 @@ private JSONObject getFilteredStatsForRollup(Rollup rollup, Set<MetricStat> filt
                 } else {
                     filteredObject.put(stat.toString(), filteredValue);
                 }
-            } catch (Exception ex) {
-                log.warn(ex.getMessage(), ex);
+            } catch (UnsupportedOperationException ex) {
+                log.warn(ex.getMessage());
```

Comment:
@usnavi , I don't think we are returning 400 in this case. They are fetching multiple metrics. They just passed in ```select=average``` for all of them. I think we just send back what submetrics are supported for each type of metrics, and ignore the rest.

I am fine with debug as well. Let's see what @ChandraAddala thinks.